### PR TITLE
Adding Parallel execution and function documentation

### PR DESCRIPTION
**Pull request by paeselhz**

### CODE REVIEW

**Comment by renanxcortes** [comment]
This is awesome! However, I'm getting a local error when trying to install from your branch. Could you try running `devtools::install_github("paeselhz/springerQuarantineBooksR")` to double-check if it works on your side? In addition, could you tweak the ReadMe file?

File: R/download_springer_book.R [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+      )
+
+    }
+
+    t2 <- Sys.time()
+
+    return_time <- t2 - t1
+
+    message(paste0('Time Elapsed: ', format(return_time, digits = 2)))
```
Comment (renanxcortes) [inline R/download_springer_book.R]: Could you please also add the book title in the message here?

File: R/download_springer_book.R [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+#' @export
+#'
+#' @examples
+#'
+#' \dontrun{
+#' download_springer_book(book = "Fundamentals of Power Electronics")
+#' }
+#'
+download_springer_book <-
+  function(book, destination_folder = 'books', springer_table = NULL) {
```
Comment (renanxcortes) [inline R/download_springer_book.R]: I think `"books"` is too generic for this project. My original idea was `"springer_books"` but I'm thinking to swap to an even more specific: `"springer_quarantine_books"` as default repo.

File: R/download_springer_book_files.R [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+  }
+
+  message("Downloading title latest editions.")
+
+  if(parallel) {
+
+    future::plan(future::multisession)
+
+    furrr::future_map(
```
Comment (renanxcortes) [inline R/download_springer_book_files.R:81]: Awesome!

**Review by renanxcortes — CHANGES_REQUESTED** [review]

File: R/download_springer_book.R (hunk 1 above)
Comment (paeselhz) [reply R/download_springer_book.R]: Sure, no problems. I'll be adding that in the next commits

File: R/download_springer_book.R (hunk 2 above)
Comment (paeselhz) [reply R/download_springer_book.R]: Ok, will be changing that in the functions as well

File: R/download_springer_book_files.R (hunk 3 above)
Comment (paeselhz) [reply R/download_springer_book_files.R:81]: Thanks man!! Looking forward to improve this!

**Comment by paeselhz** [comment]
> This is awesome! I'm getting a local error when trying to install from your branch. Could you try running `devtools::install_github("paeselhz/springerQuarantineBooksR")` to double-check if it works on your side? In addition, could you tweak the ReadMe file?

I've tried Running the devtools installation, and it apparently is working on Unix-like machines only... Will be working further to allow integration with Windows machines as well. Also, will be tweaking with the ReadMe

**Review by renanxcortes — APPROVED** [review]

**Comment by renanxcortes** [comment]
Absolutely amazing! Thanks for this refactorization, @paeselhz !

**Comment by renanxcortes** [comment]
Most of the changes in this PR were reverted (but not the PR itself) since a bug appeared (https://github.com/renanxcortes/springerQuarantineBooksR/issues/6).

**Pull request merged**